pkg/executor/executortype/poolmgr/gp.go

```diff
@@ -387,10 +388,32 @@ func (gp *GenericPool) specializePod(ctx context.Context, pod *apiv1.Pod, fn *fv
 	if len(podIP) == 0 {
 		return errors.Errorf("Pod %s in namespace %s has no IP", pod.ObjectMeta.Name, pod.ObjectMeta.Namespace)
 	}
+	for _, cm := range fn.Spec.ConfigMaps {
+		_, err := gp.kubernetesClient.CoreV1().ConfigMaps(gp.fnNamespace).Get(ctx, cm.Name, metav1.GetOptions{})
+		if err != nil && k8s_err.IsNotFound(err) {
```

Comment:
Please modify this condition. Do same with secret as well.
If err != nil {
    If k8s_err.IsNotFound(err) {
        // log error specific to configmap not found
    } else {
        // log generic error got from kubernetes server
    }
}

Reply:
Done